<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>430</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>837</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>837</width>
    <height>430</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>正方教务系统刷课软件 by.Syhen（喝药）</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QListWidget" name="lw_course">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>71</y>
     <width>181</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_gxk">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>41</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>公选课</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_delete">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>340</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_close">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_bzy">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>41</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>本专业选课</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="b_down">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>310</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>下移</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lw_msg">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>71</y>
     <width>631</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="lw_choose">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>281</y>
     <width>771</width>
     <height>111</height>
    </rect>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="b_ok">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>401</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>开始刷课</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_up">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>280</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>上移</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_clear">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>370</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>403</y>
     <width>421</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="b_stop">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>401</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb_id">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>8</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb_pw">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;密码&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="b_login">
   <property name="geometry">
    <rect>
     <x>599</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="code">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Shen/Anaconda2/Lib/site-packages/pytesser/1vy4.gif</normaloff>C:/Users/Shen/Anaconda2/Lib/site-packages/pytesser/1vy4.gif</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>78</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="l_zt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>403</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>状态：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="b_code">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>40</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>14</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入正方教务系统登录界面网址：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="url">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>11</y>
     <width>631</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>41</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>学号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>42</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>356</x>
     <y>41</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb_name">
   <property name="geometry">
    <rect>
     <x>383</x>
     <y>40</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>url</tabstop>
  <tabstop>tb_id</tabstop>
  <tabstop>tb_pw</tabstop>
  <tabstop>tb_name</tabstop>
  <tabstop>code</tabstop>
  <tabstop>b_code</tabstop>
  <tabstop>b_login</tabstop>
  <tabstop>rb_bzy</tabstop>
  <tabstop>rb_gxk</tabstop>
  <tabstop>lw_course</tabstop>
  <tabstop>lw_msg</tabstop>
  <tabstop>lw_choose</tabstop>
  <tabstop>b_up</tabstop>
  <tabstop>b_down</tabstop>
  <tabstop>b_delete</tabstop>
  <tabstop>b_clear</tabstop>
  <tabstop>b_ok</tabstop>
  <tabstop>b_stop</tabstop>
  <tabstop>b_close</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>b_close</sender>
   <signal>clicked()</signal>
   <receiver>dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_clear</sender>
   <signal>clicked()</signal>
   <receiver>lw_choose</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>805</x>
     <y>383</y>
    </hint>
    <hint type="destinationlabel">
     <x>726</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
